dogecoin/configuration.go

```diff
@@ -69,9 +55,37 @@ const (
 	allFilePermissions = 0777
 )
 
+var (
+	// configurationDirectory is the configuration path prefix
+	configurationDirectory string = os.Getenv("CONFIG_DIR")
+
+	// mainnetConfigPath is the path of the Bitcoin
+	// configuration file for mainnet.
+	mainnetConfigPath string
+
+	// testnetConfigPath is the path of the Bitcoin
+	// configuration file for testnet.
+	testnetConfigPath string
+
+	testnetTransactionDictionary string
+	mainnetTransactionDictionary string
+
+	// DataDirectory is the default location for all
+	// persistent data.
+	DataDirectory string = os.Getenv("DATA_DIR")
+)
+
 // LoadConfiguration attempts to create a new Configuration
 // using the ENVs in the environment.
 func LoadConfiguration(baseDirectory string) (*configuration.Configuration, error) {
+	if configurationDirectory == "" {
+		configurationDirectory = "/app"
+	}
```

Comment:
I think empty string check should be done here consistently with the comment on `main.go`

Reply:
I'm actually running into some errors using len(dir) != 0 and len(dir) == 0. Can you and @solo-fish explain why this is preferred method of checking an empty string? I was reading a thread on this topic and one user found negligible performance benefits of about .01 ns/op. Not to get into syntactic preference per dev or nit as I am understanding but I'm honestly wondering why this is not recommended.

Reply:
To me it's just about consistency. If we change it at one place, we can change it at the other.

Reply:
For some reason after I switched to checking by length I was failing the configuration_test.go config_path so I added another argument to check whether or not it was testing as seen on line 82 in dogecoin/configuration.go. Not sure how I passed tests before but I've passed all tests and built a container that ran successfully. <3

Reply:
@xanimo Hey! Sorry for the delayed response - @patricklodder is correct on the standard, however,  you should consider using `os` from the stdlib. I could follow up on this as well with a PR.

Excerpt from the documentation
> Package os provides a platform-independent interface to operating system functionality. The design is Unix-like, although the error handling is Go-like; failing calls return values of type error rather than error numbers.

- https://golang.org/pkg/os